middle_end/HACKING.adoc

```diff
@@ -0,0 +1,70 @@
+== Getting a grasp of the Flambda
+
+Flambda is an intermediate representation generated from Lambda by
```

Comment:
You should probably use code quoting for module names (I was confused at the beginning for what was a noun and what was a module name ...).